data/json/recipes/recipe_food.json

```diff
@@ -4115,6 +4116,38 @@
     "tools": [ [ [ "surface_heat", 3, "LIST" ] ], [ [ "rock_quern", -1 ], [ "clay_quern", -1 ] ] ],
     "components": [ [ [ "chili_pepper", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "result": "garlic_powder",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "45 m",
```

Comment:
Doesn't it take 3-8 hours to dry?

Reply:
does it? I could alter the recipe then. I'm not sure people will be happy that it takes so long so I'll make it between 1-2 hours.

Reply:
after looking it up, it seems that is for a massive amount of garlic. the 45 minutes is for a single clove. I'm sure if you're dehydrating several bulbs worth of garlic it'd take 3-8 hours ingame.

Reply:
It just seems that in every recipe I saw there is a long time for passive dehydration. For example, here it says "4-6 hours in oven, or 2-3 days in open air"

https://www.alphafoodie.com/how-to-make-garlic-powder/

Reply:
huh. I assumed the player was just speed-dehydrating the garlic if that even exists. Similar to how you dehydrate fish or meat within 20 minutes ingame.

Reply:
I'm, going to resolve this conversation if that's okay with you, it seems like this isn't just the powder but dehydrated foods as a whole.